editor.planx.uk/src/components/Header.tsx

```diff
@@ -428,9 +433,25 @@ const PublicToolbar: React.FC<{
 
 const ServiceTitle: React.FC = () => {
   const flowName = useStore((state) => state.flowName);
+  const route = useCurrentRoute();
+  const path = route.url.pathname.split("/").slice(-1)[0];
 
   return (
     <ServiceTitleRoot data-testid="service-title">
+      {(path === "preview" || path === "draft") && (
+        <Chip
+          label={capitalize(path)}
+          variant="notApplicableTag"
+          size="medium"
+          icon={
+            path === "preview" ? (
+              <OpenInNewIcon fontSize="small" />
+            ) : (
+              <OpenInNewOffIcon fontSize="small" />
+            )
+          }
```

Comment:
```suggestion
          icon={{
            preview: <OpenInNewIcon fontSize="small" />
            draft: <OpenInNewOffIcon fontSize="small" />
          }[path]}
          }
```

Maybe a little clearer and more expandable?

Reply:
Thanks @DafyddLlyr , I see a recurring theme here — my first instinct to use boolean / ternary operators vs expandable list based code.